packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -335,13 +335,13 @@ function attSubnetLabel(subnet: SubnetID): string {
 function getMetricsTopicStrToLabel(config: BeaconConfig, opts: {disableLightClientServer: boolean}): TopicStrToLabel {
   const metricsTopicStrToLabel = new Map<TopicStr, TopicLabel>();
 
-  for (const {name: fork} of config.forksAscendingEpochOrder) {
-    const topics = getCoreTopicsAtFork(config, fork, {
+  for (const boundary of config.forkBoundariesAscendingEpochOrder) {
```

Comment:
When there are two boundaries (a hard fork and a bpo fork) have the same epoch, we should ignore hard fork, or else `stringifyGossipTopic` will throw error because fork digest is not found.

Need some logic similar to this
```
    if (currentEpoch === nextEpoch) {
      continue;
    }
```